src/taskFunctionality.js

```diff
@@ -0,0 +1,49 @@
+import { taskArray, arr } from './declarations.js';
+import Task from './tasks.js';
+
+const addTask = (addToList) => {
+  if (addToList.value !== '') {
+    const task = new Task(addToList.value);
+    taskArray.push(task);
+    localStorage.setItem('task', JSON.stringify(taskArray));
+    addToList.value = '';
+    window.location.reload(true);
+  }
+};
+
+const deleteTask = (j) => {
+  document.querySelectorAll('.checkbox').forEach((check) => {
+    if (check.checked === false) {
+      arr.push(check.nextElementSibling.innerText);
+    }
+  });
+  const newtaskArray = taskArray.filter((item) => arr.includes(item.description));
+  newtaskArray.forEach((task) => {
+  // eslint-disable-next-line no-plusplus
+    task.index = j++;
```

Comment:
### Recommended suggestion
- [ ] Kindly enable the eslint, you can use `j += 1` in place of  `j++`  😄